rubyUtils.gradle

```diff
@@ -151,17 +151,24 @@ void buildGem(File projectDir, File buildDir, String gemspec) {
  * @param projectDir Gradle projectDir
  * @param buildDir Gradle buildDir
  * @param plugin Plugin to run specs for
- * @param args CLI arguments to pass to rspec
+ * @param args Optional arguments to pass to the rake task
  */
-void rake(File projectDir, File buildDir, String task) {
+void rake(File projectDir, File buildDir, String task, String... args) {
     executeJruby projectDir, buildDir, { ScriptingContainer jruby ->
         jruby.currentDirectory = projectDir
         jruby.runScriptlet("require 'rake'; require 'time'")
+        def rakeArgs = args ? "'${args.join("','")}'" : ""
         jruby.runScriptlet("""
-                rake = Rake.application
-                rake.init
-                rake.load_rakefile
-                rake['${task}'].invoke
+                begin
+                    rake = Rake.application
+                    rake.init
+                    rake.load_rakefile
+                    rake['${task}'].invoke(${rakeArgs})
+                rescue => e
+                    puts "Rake task error: #{e.class}: #{e.message}"
+                    puts "Backtrace: #{e.backtrace.join("\\n")}"
+                    raise e
+                end
```

Comment:
Respect to `8.17` backport here is just a missed new line https://github.com/elastic/logstash/pull/17315/files#diff-7a2156b270dee259f243cf6371cdeb28e3bfdbc8c0e418cb37cfb4724144b305R172

Reply:
Good eye. This had some merge conflicts that needed some special care. I must have slipped up on a whitespace change.